test/test_sparse_csr.py

```diff
@@ -363,23 +365,29 @@ def test_print(self, layout, device):
             self.maxDiff = orig_maxDiff
             raise
 
+    def _smallest_divisor(self, n):
+        for i in range(2, int(n ** 0.5) + 1):
+            if n % i == 0:
+                return i
+        return n
+
     @skipMeta
     @all_sparse_compressed_layouts()
     @dtypes(*all_types_and_complex_and(torch.half, torch.bool, torch.bfloat16))
     def test_copy(self, layout, device, dtype):
 
         def run_test(shape, nnz, index_type):
-            block_size = (2, 3) if layout in {torch.sparse_bsr, torch.sparse_bsc} else ()
+            blocksize = tuple(map(self._smallest_divisor, shape[-2:])) if layout in {torch.sparse_bsr, torch.sparse_bsc} else ()
```

Comment:
Why not hardcode the values instead of writing this function? I agree that it's cleaner, but for tests transparency matters most (to me at least).

Reply:
It would just mean that `run_test` function would need to be extended with `blocksize` argument. Makes sense to me.

Reply:
Fixed.